### DISCUSSION TITLE
除了wss之外，是不是还可以支持纯https？（就是不需要websocket，直接用http来传输）

### DISCUSSION THREAD

**Discussion by yicixingzhanghao001** [v2fly/v2ray-core#612] [Ideas]
大家好。我一直使用wss再由百度加速中转，但是百度最近严查wss所以在这么做就无效了。

如果v2ray支持纯https的话，我可以继续像以前一样这样中转。

我现在用https://github.com/WarwolfSystem 可以用https做socks5，但是他们的代码不更新了。

**Comment by okudayukiko** [comment]
沒用的，因為在中轉端也會將TLS解密。
